querydsl-postgrest/src/main/java/fr/ouestfrance/querydsl/postgrest/services/BulkExecutorService.java

```diff
@@ -0,0 +1,53 @@
+package fr.ouestfrance.querydsl.postgrest.services;
+
+import fr.ouestfrance.querydsl.postgrest.annotations.PostgrestConfiguration;
+import fr.ouestfrance.querydsl.postgrest.model.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ * Bulk Executor service allow to execute on a wide range of data
+ * Options allow you to specify bulk parameters :
+ * - countOnly : Do not get object representation
+ * - pageSize: Specify chunk size, allow to iterate on postgrest and avoid timeout exceptions
+ */
+public class BulkExecutorService {
+
+    public <T> BulkResponse<T> execute(Function<BulkRequest, BulkResponse<T>> function, BulkRequest request, BulkOptions options) {
+        List<String> prefers = request.getHeaders().computeIfAbsent(Prefer.HEADER, x -> new ArrayList<>());
+        if (options.isCountOnly()) {
+            prefers.stream().filter(x -> x.startsWith("return")).findFirst().ifPresent(prefers::remove);
+            prefers.add(Prefer.Return.HEADERS_ONLY);
+        }
+        // Add default count
+        prefers.add("count=" + PostgrestConfiguration.CountType.EXACT.name().toLowerCase());
+        // If we want to bulk update (multiple page)
+        Pageable pageable = null;
```

Comment:
Add sort - function

Reply:
Finally, dont add sort functions, because it's possible to put Pageable on PATCH / DELETE